backend/routes/auth.js

```diff
@@ -128,4 +137,121 @@ router.post(
   }
 );
 
+/**
+ * @method - POST
+ * @param - /forgot
+ * @description - Saves user's forgot password token in DB and sends user email
+ */
+router.post(
+  "/forgot",
+  [check("email", "Please enter a valid email").not().isEmpty().isEmail()],
+  async (req, res) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      return res.status(400).json({
+        message: errors.errors[0].msg,
+      });
+    }
+
+    const { email } = req.body;
+
+    try {
+      let user = await User.findOne({
+        email,
+      });
+
+      if (!user) {
+        return res.status(400).json({
+          message: "Invalid email! Please try again.",
+        });
+      }
+
+      const randomString = Math.random().toString(36).slice(2);
+      // Save user's forgot password token
+      user.forgotPasswordToken = randomString;
+      await user.save();
+
+      // Send the forgot password email to the user
+      const link = process.env.FRONTEND_URL + "reset/" + randomString;
+      const options = {
+        from: "TSE_Supply_Chain@outlook.com",
+        to: user.email,
+        subject: "Reset Password",
+        text:
+          "This email is being sent to you because you have forgotten your password. \n\n Click the link below to reset your password. \n " +
+          link +
+          "\n \nIf the link is not working, copy and paste it into the address bar.",
```

Comment:
```suggestion
          "This email is being sent to you because you have forgotten your password.\n\nClick the link below to reset your password.\n" +
          link +
          "\n\nIf the link is not working, copy and paste it into the address bar.",
```

Reply:
Alternatively, you could create an array of lines and use `.join("\n\n")` or similar. It'll accomplish the same thing, it's just a question of style.